projects/step-frontend/src/lib/modules/scheduler/execution-task-parameters-list/execution-task-parameters-list.component.scss

```diff
@@ -0,0 +1,47 @@
+.nav-to-plan-icon {
+  transform: scale(0.8);
+  cursor: pointer;
+}
+
+.status {
+  color: black;
+  padding: 5px;
+  border-radius: 3px;
+  cursor: pointer;
+  border-width: 1px;
+  border-style: solid;
+  &:hover {
+    border-style: none;
+  }
+}
+
+.status-on {
+  background-color: #35c163;
+  border-color: #35c163;
+}
+
+.status-off {
+  background-color: #d90606;
+  border-color: #d90606;
+}
+
+.cell-actions {
+  width: 20rem;
+
+  button {
+    padding: 0;
+    min-width: 4rem;
+    min-height: 4rem;
+  }
+}
+
+.users-header {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  display: flex;
+  justify-content: flex-end;
+
+  step-btn-group {
+    margin-left: 1rem;
+  }
+}
```

Comment:
This is not the users component, but because I saw this mistake multiple times I renamed the class from "COMPONENT-header" to just "header", we should do the same here. (Probably we can put much code somewhere global later on)